js/components/dashboard/activity-questions.js

```diff
@@ -19,6 +21,10 @@ export default class ActivityQuestions extends PureComponent {
             expanded && activity.get('questions').filter(q => q.get('visible')).map(q => {
               const questionIsExpaned = expandedQuestions.get(q.get('id').toString())
               const expanded = showFullPrompts || questionIsExpaned
+              const openQuestionDetails = (e) => {
+                e.stopPropagation()
```

Comment:
Is that to prevent question collapsing? If so, some comment could be nice so someone doesn't remove it by accident (and tests that would cover it is perhaps tricky).